### PR TITLE
Created Join the Community Section

### LINKED ISSUE #3: Create Join the Community Section
Here's what you will be building:
Join the Community

Please refer to the [design](https://www.figma.com/proto/ZDkwUTcrWZvv67wCdnt6Xu/freeCodeCamp-Manila?node-id=47%3A1&scaling=scale-down-width), and make sure you read the [project plan](https://github.com/fccManila/fccmanila.github.io/blob/develop/PROJECT_PLAN.md).

If you have any questions, don't hesitate to ask 😄

### PR DESCRIPTION
**Pull request by engrleir**
**Description**

Created the 'Join the Community Section' as closely as I can based on the [design](https://www.figma.com/proto/ZDkwUTcrWZvv67wCdnt6Xu/freeCodeCamp-Manila?node-id=47%3A1&scaling=scale-down-width).

**Screenshots**

_Before_
Screen Shot 2019-10-05 at 2 07 16 PM

_After_
Screen Shot 2019-10-05 at 2 07 05 PM

**Notes**

Implementation not yet responsive. Please advise if mobile design prototype is available.

Closes #3

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -1,75 +1,116 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto&display=swap");
```
Comment (jromest) [inline src/css/style.css]: You don't need to add this, font are available in `index.html` under a `<link>` tag in head.

File: src/index.html [hunk 2]
```diff
@@ -1,50 +1,70 @@
...
+
+  <section class="community">
+    <h2 class="community-title">Join the Community</h2>
+    <p class="community-text">
+      Engage in a friendly community of campers on our Facebook Group. Anyone Interested in learning
+      how to code is welcome, regardless of your current level of knowledge. Attend our monthly
+      meetups and stay accountable, get help, and learn together.
+    </p>
+    <div class="community-btn">
+      <a href="https://facebook.com/groups/free.code.camp.manila/" target="blank">Join our Facebook Group</a>
```
Comment (jromest) [inline src/index.html]: Please change `target="blank"` to target="_blank". Also add `rel="noopener noreferrer"` [here's why?](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)

**Comment by engrleir** [comment]
Requested changes has been implemented. Thanks. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jromest** [comment]
✨ LGTM ✨

Thanks, @engrleir! Happy contributing.

**Pull request merged**